.travis.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         - npm install -g cspell
       script:
         - pip3 install -r requirements-dev.txt
-        - tox -e style
```

Comment:
Just a comment. I personally would have preferred style because its easier to read

Reply:
Ah yeah that's right. I just tried to mimic what `tox` does, like `py` for the main test environment